routes/discover.js

```diff
@@ -17,11 +17,14 @@ exports.getOrganizations = async function (req, res) {
   let filtered;
 
   if (filter === 'all') {
-    filtered = await database.getAllOrganizations(filter);
+    filtered = await database.getAllOrganizations(req.user);
   } else {
-    filtered = await database.getFilteredOrganizations(filter);
+    filtered = await database.getFilteredOrganizations(filter, req.user);
   }
```

Comment:
nit: this if/else block can be updated to a ternary: 
`const filtered = filter === 'all' ? await database.getAllOrganizations(req.user) : await database.getFilteredOrganizations(filter, req.user); `